sidebars.js

```diff
@@ -94,11 +94,15 @@ module.exports = {
         "node-operators/cli-reference"
       ],
     },
-
     {
       type: "category",
-      label: "Exchange Operators",
-      items: ["exchange-operators/exchange-faq"],
+      label: "Node Developers",
+      items: [
+        "node-developers/overview",
```

Comment:
@iregina appears to be a typo, missing a closing `}` for Node Developers

Reply:
Fixed! Thanks.